tests/unit/fake_data_root/sos_commands/dpkg/dpkg_-l

```diff
@@ -947,7 +947,7 @@ ii  ssh-import-id                          5.5-0ubuntu1
 ii  strace                                 4.11-1ubuntu3                                   amd64        System call tracer
 ii  sudo                                   1.8.16-0ubuntu1.10                              amd64        Provide limited super user privileges to specific users
 ii  sysstat                                11.2.0-1ubuntu0.3                               amd64        system performance tools for Linux
-ii  systemd                                229-4ubuntu21.28                                amd64        system and service manager
+ii  systemd                                237-3ubuntu10.46                                amd64        system and service manager
 ii  systemd-sysv                           229-4ubuntu21.28                                amd64        system and service manager - SysV links
```

Comment:
can you make all the systemd* packages the same version

Reply:
done.